### ISSUE TITLE
Manual invocation of tests/tests.sh fails

### ISSUE THREAD

**Issue by nopdotcom** [StreisandEffect/streisand#1052]
### Expected behavior:

```
The `test/test.sh` wrapper script is invoked to perform tests and supports
an argument for specifying what actions should be taken. The following are
supported arguments:
```

`./tests/tests.sh syntax` checks syntax.

### Actual Behavior:

It looks like something really bad is happening with `SITE`:

```
+ ansible-playbook -i /Users/nop/nop-streisand/tests/inventory --extra-vars=@global_vars/vars.yml --extra-vars=@ /Users/nop/nop-streisand/tests/development-setup.yml
```

Note the extra space after the second `--extra-vars`.

Log follows.

```
$ tests/tests.sh syntax

  S T R E I S A N D

ansible-playbook 2.4.0.0
  config file = /Users/nop/nop-streisand/tests/ansible.cfg
  configured module search path = [u'/Users/nop/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python2.7/site-packages/ansible
  executable location = /usr/local/bin/ansible-playbook
  python version = 2.7.14 (default, Sep 25 2017, 09:53:17) [GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.42.1)]
Using /Users/nop/nop-streisand/tests/ansible.cfg as config file
 [WARNING]: Found both group and host with same name: streisand

 [WARNING]: Found both group and host with same name: localhost

ERROR! Unable to retrieve file contents
Could not find or access '/Users/nop/nop-streisand'
$
```
### Steps to Reproduce:
1. Check out `tests/tests.sh` at revision d82cd54695168f2a625f789149ee824df517ac79 or later
1. Run any tests subcommand 

##### *Operating System of client:*
Ubuntu 16.04 LTS, macOS
##### *Version of Ansible, using `ansible --version` :*

ansible 2.4.1.0
ansible 2.4.0.0

##### *Output from `git rev-parse HEAD` in your Streisand directory :*

How about a git bisect?

git bisect start
* good: [b197e7bd79ca1bc7b65b05dae778c82804ba06e0] Do not show "changed" when running RO commands (#903)
git bisect good b197e7bd79ca1bc7b65b05dae778c82804ba06e0
* bad: [ca2106e73db72209c360dc6912acddfa76b5c6b8] Use correct SSH keys metadata for GCE provisioning. (#1048)
git bisect bad ca2106e73db72209c360dc6912acddfa76b5c6b8
* good: [0272b14f38c40df4b8af33691e09711176b3a4b1] Sentimentally update the README in preparation for the repo move.
git bisect good 0272b14f38c40df4b8af33691e09711176b3a4b1
* bad: [07538d088a16b2059a2be9e0ad324187d4051fbb] Merge pull request #1019 from alimakki/readme_azure
git bisect bad 07538d088a16b2059a2be9e0ad324187d4051fbb
* good: [25a915a01b7ec93cdb3486935271a8a1cc2f8ccf] README-fr update (#1004)
git bisect good 25a915a01b7ec93cdb3486935271a8a1cc2f8ccf
* bad: [fc0cca533af5efdb69b73c900cb31398067c94d4] Refactor OpenSSL CA/Server/Client certificate generation into its own role (#999)
git bisect bad fc0cca533af5efdb69b73c900cb31398067c94d4
* bad: [f8394b55c8b91094cba191878f148934c580fd5b] Only generate Mirror page when there is mirrored software. (#1003)
git bisect bad f8394b55c8b91094cba191878f148934c580fd5b
* bad: [d82cd54695168f2a625f789149ee824df517ac79] Modular Streisand: improved test coverage of enabled/disabled services (#988)
git bisect bad d82cd54695168f2a625f789149ee824df517ac79
* first bad commit: [d82cd54695168f2a625f789149ee824df517ac79] Modular Streisand: improved test coverage of enabled/disabled services (#988)

ca2106e73db72209c360dc6912acddfa76b5c6b8

**Comment by cpu** [comment]
Definitely seems like a regression!

Since you've put some time into identifying a likely commit that introduced it & you're probably the only person using this script outside of CI, do you want to take a crack at fixing it?

**Comment by nopdotcom** [comment]
That sounds fair to me!

**Comment by cpu** [comment]
Resolved with #1056
